tools/printcheck/src/printcheck/checks.py

```
@@ -162,21 +172,136 @@ def check_integrity(mesh: trimesh.Trimesh, cfg: Config):
+      across its width, so the test is a morphological erosion by half the
+      bridge width: if the footprint erodes to nothing it is everywhere
+      narrower than `bridge_max_mm`. A ring erodes away here even though it is
+      geometrically wide — which is why a plain inradius/erosion test alone was
+      chosen — but that same erosion wrongly clears a *solid* acute blob (an
+      equilateral triangle of side 8 mm has inradius 2.31 < 2.5 mm yet its
+      shortest straight bridge is 6.9 mm).
+
+    - A solid/compact region must be crossed by full straight bridges, so the
+      binding span is its convex-hull minimum width, not its local thinness.
+
+    The two are told apart by Polsby–Popper compactness (4π·area / perimeter²):
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)